job_templates/snippet/property_parameter-definition_common.xml.em

```diff
@@ -124,7 +124,7 @@ choices.remove(cmake_build_type)
           <choices class="java.util.Arrays$ArrayList">
             <a class="string-array">
               <string>2019</string>
-              <string>2017</string>
+              <string>2022</string>
```

Comment:
Is this parameter actually used anywhere with the docker/cinc windows agents?

When we had bare metal agents both versions of visual studio would be co-installed and we'd use this parameter to select which version's copy of vcvarsall.bat to invoke.

If this is no longer used then we probably shouldn't present it.

Reply:
I'm using that parameter to handle the vs invocation in the docker file and being able to override the vs version at runtime. See

https://github.com/ros2/ci/blob/20087f40800a9e5f6f427b4659a2d067f2eec425/windows_docker_resources/Dockerfile#L72

Ideally the goal is to restore the functionality to select VS version to CI.